### ISSUE TITLE
Add Litematica and MaLiLib dependencies to fabric.mod.json

### ISSUE THREAD

**Issue by Poopooracoocoo** [aleksilassila/litematica-printer#206]

**Comment by douira** [comment]
Yes I would like to add that this is important. Not declaring the dependency on `litematics` means people attempt to use this mod and run into errors without being notified of the fact that they forgot to install litematica.

**Comment by Poopooracoocoo** [comment]
This has been fixed in newer versions of Litematica Printer.
